crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -444,6 +445,7 @@ impl CheckpointBuilder {
         notify_aggregator: Arc<Notify>,
         metrics: Arc<CheckpointMetrics>,
         max_transactions_per_checkpoint: usize,
+        max_checkpoint_size: usize,
```

Comment:
nit: s/max_checkpoint_size/max_checkpoint_size_bytes/?

Reply:
I chose this name to match the same name in `ProtocolConfig`, which was chosen to match existing `max_tx_size` (for which the unit is also bytes). I had thought about adding `_bytes` but figured it was clear enough already. However, if you are asking for it, maybe it's not clear enough! is it ok if I stay consistent with existing in this PR, and then change together in a follow-up?

Reply:
Sounds good!